tests/scripts/thread-cert/Cert_5_1_07_MaxChildCount.py

```diff
@@ -150,6 +167,48 @@ def test(self):
                     self.assertTrue(self.nodes[LEADER].ping(addr, size=106))
                     break
 
+    def verify(self, pv: PacketVerifier):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        ROUTER = pv.vars['ROUTER']
+        router_pkts = pkts.filter_wpan_src64(ROUTER)
+
+        # Step 1: The DUT MUST send properly formatted MLE Parent Response
+        #         and MLE Child ID Response to each child.
+        for i in range(1, 7):
+            _pkts = router_pkts.copy().filter_wpan_dst64(pv.vars['SED%d' % i])
```

Comment:
```suggestion
            _pkts = router_pkts.filter_wpan_dst64(pv.vars['SED%d' % i], cascade=False)
```
We can use `cascade=False` to elide `.copy()`